src/modules/physics/box2d/wrap_Physics.cpp

```diff
@@ -477,12 +477,100 @@ int w_setMeter(lua_State *L)
 	return 0;
 
 }
+
 int w_getMeter(lua_State *L)
 {
 	lua_pushinteger(L, Physics::getMeter());
 	return 1;
 }
 
+int w_linearStiffness(lua_State* L) {
```

Comment:
love's typical function naming convention is to have a verb at the start of the function name - in this case I think `compute` makes sense, e.g. `computeLinearStiffness`, etc.

That convention should be applied to the C++ code as well as the Lua APIs.